finesse/em27_status.py

```diff
@@ -0,0 +1,22 @@
+"""Provides an enum representing the EM27's state."""
+from enum import Enum
+
+
+class EM27Status(Enum):
+    """The state of the EM27 interferometer.
+
+    These values are taken from the manual.
+    """
+
+    IDLE = 0
+    CONNECTING = 1
+    CONNECTED = 2
+    MEASURING = 3
+    FINISHING = 4
+    CANCELLING = 5
+    UNDEFINED = 6
+
+    @property
+    def connected(self) -> bool:
```

Comment:
While it is clearly different than `CONNECTED` as it is lower case, it is the same word, which might be confusing when reading the code. I would use `is_connected`, which will give a better sense of this being a flag.